.github/workflows/community-review.sh

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+#
+# Community review GitHub Action - sets the community review labels on PRs to show case
+# community review activity. See Flip-518 for details
+set -e
+
+# =============================================================================
+# Global variables
+# =============================================================================
+REPO_OWNER=apache
+REPO_NAME=flink
+LGTM_LABEL="community-reviewed-LGTM"
+COMMUNITY_REVIEW_LABEL="community-reviewed"
+USER_CACHE_FILENAME="user_cache.txt"
```

Comment:
Are we sure the script runner user will have permission to create a file in the workdir root? Previously in other projects I had trouble writing files on GH runner nodes, but `/tmp` was safe. If this works, I have nothing against it, just wanted to ask if are we sure.

Reply:
the script runner will be running under the github token from the action, which has write permission. So we should be good but I will double check.

Reply:
Looks like it has permission. I suspect the `/tmp` was when github action did not have write access permission.

Reply:
Correct, that was a GH enterprise runner, maybe quite strict permissions, so I think it's good to leave it as is if it works, cause this is the simplest solution.